frontend/packages/console-shared/src/components/formik-fields/CodeEditorField.scss

```diff
@@ -2,36 +2,18 @@
 @import '../../../../../public/style/vars';
 
 .osc-yaml-editor {
-  margin-right: -$pf-v6-global-gutter;
-  margin-top: -$pf-v6-global-gutter--md;
-
-  @media (max-width: $pf-v6-global--breakpoint--md) {
-    margin-left: -$pf-v6-global-gutter;
-  }
-  @media (min-width: $pf-v6-global--breakpoint--xl) {
-    margin-right: -$pf-v6-global-gutter--md;
-  }
-
-  .co-p-has-sidebar__body {
-    flex-direction: column;
-    display: flex;
-  }
+  margin-right: calc($pf-v6-global-gutter-x * -1);
```

Comment:
Note I removed the gutter negation for mobile because we don't do that elsewhere.  I'd really like us to consolidate to a single editor instance.